STM32Cube/Tasks/sdmmc.c

```diff
@@ -0,0 +1,27 @@
+#include "sdmmc.h"
+
+const char *logsPath = "/LOGS";
+char logFileName[LOG_FILE_NAME_SIZE];
+
+static int computeFolderSize(const char *path) {
```

Comment:
adding a few quick comments describing these methods would be very helpful in the future if anyone has to touch this file.. i was so confused cuz "computeFolderSize" sounds like ur counting the size in bytes or smth but its actually just counting files?